cockatrice/src/tab_supervisor.cpp

```diff
@@ -509,17 +509,30 @@ void TabSupervisor::processUserMessageEvent(const Event_UserMessage &event)
     TabMessage *tab = messageTabs.value(senderName);
     if (!tab)
         tab = messageTabs.value(QString::fromStdString(event.receiver_name()));
-    if (!tab) {
+
+    if (!tab)
+    {
```

Comment:
Hmm, didn't know. Thanks for the notice.

I'm personally a fan of braces on a new line by themselves, but if I have to follow the format I guess I will